packages/angular-meteor-data/tests/integration/reactive.spec.js

```diff
@@ -388,5 +460,35 @@ describe('angular-meteor.reactive', function() {
         expect(changedSpy).toHaveBeenCalled();
       });
     });
+
+    describe('$reactivateCollection()', function() {
+      var scope;
+      var vm;
+
+      beforeEach(function() {
+        scope = $rootScope.$new();
+        vm = scope.$viewModel({});
+      });
+
+      afterEach(function() {
+        scope.$destroy();
+      });
+
+      it('should return model', function() {
+        vm.myProp = 10;
```

Comment:
This should probably be an array because `$watchCollection` expects a collection, not an object.